app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -180,6 +201,78 @@ public static void showOdkCollect(
         callingActivity.startActivityForResult(intent, requestCode);
     }
 
+    private static List<OpenMrsXformIndexEntry> getLocalFormEntries() {
+        List<OpenMrsXformIndexEntry> entries = new ArrayList<>();
+
+        final ContentResolver resolver = App.getInstance().getContentResolver();
+        Cursor c = resolver.query(Contracts.Forms.CONTENT_URI, null, null, null, null);
```

Comment:
It's generally better to specify the projection (the columns) for database reads, in case the database schema changes in the future.